Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN apk update && apk upgrade && apk add curl && apk add openjdk17
 
 COPY ./build/libs/mal-cli-1.0-all.jar ./
 
-ENTRYPOINT ["java", "-jar", "./mal-cli-1.0-all.jar"]
+# Port used for redirect during login flow
+EXPOSE 8080
```

Comment:
this seems to still not work on my laptop for work, seems due to certificate issues